<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransferConfirmDialog</class>
 <widget class="QDialog" name="TransferConfirmDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>543</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Microsoft YaHei UI Light</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>543</height>
    </rect>
   </property>
   <widget class="QWidget" name="containerWidget" native="true">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>111</y>
      <width>480</width>
      <height>320</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI Light</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <widget class="QLineEdit" name="pwdLineEdit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>195</y>
       <width>260</width>
       <height>30</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="tipLabel2">
     <property name="geometry">
      <rect>
       <x>148</x>
       <y>230</y>
       <width>200</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QToolButton" name="okBtn">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>260</y>
       <width>96</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>65</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI Light</family>
       <pointsize>-1</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(52,37,90);
font: 12px &quot;Microsoft YaHei UI Light&quot;;</string>
     </property>
     <property name="text">
      <string>发往地址:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>195</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(52,37,90);
font: 12px &quot;Microsoft YaHei UI Light&quot;;</string>
     </property>
     <property name="text">
      <string>钱包密码</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>160</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI Light</family>
       <pointsize>-1</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(52,37,90);
font: 12px &quot;Microsoft YaHei UI Light&quot;;</string>
     </property>
     <property name="text">
      <string>备注:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="feeLabel">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>125</y>
       <width>241</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(51, 51, 51);
font: 13px &quot;Microsoft YaHei UI Light&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>95</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI Light</family>
       <pointsize>-1</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(52,37,90);
font: 12px &quot;Microsoft YaHei UI Light&quot;;</string>
     </property>
     <property name="text">
      <string>转账金额:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="remarkLabel">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>160</y>
       <width>260</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(52,37,90);
font: 12px &quot;Microsoft YaHei UI Light&quot;;</string>
     </property>
     <property name="text">
      <string>无</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>-1</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14px &quot;微软雅黑&quot;;
color: rgb(52,37,90);</string>
     </property>
     <property name="text">
      <string>账单确认</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QToolButton" name="cancelBtn">
     <property name="geometry">
      <rect>
       <x>254</x>
       <y>260</y>
       <width>96</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>125</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI Light</family>
       <pointsize>-1</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(52,37,90);
font: 12px &quot;Microsoft YaHei UI Light&quot;;</string>
     </property>
     <property name="text">
      <string>手续费:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="addressLabel">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>65</y>
       <width>331</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{color: rgb(52,37,90);
font: 12px &quot;Microsoft YaHei UI Light&quot;;}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
    <widget class="QLabel" name="amountLabel">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>95</y>
       <width>241</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(51, 51, 51);
font: 13px &quot;Microsoft YaHei UI Light&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QToolButton" name="closeBtn">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>20</y>
       <width>13</width>
       <height>13</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
